cli/config.go

```diff
@@ -0,0 +1,104 @@
+package cli
+
+import (
+	"os"
+	"strings"
+
+	"github.com/flashbots/mev-boost/server/types"
+	"gopkg.in/yaml.v3"
+)
+
+type RelayConfigYAML struct {
+	URL                  string `yaml:"url"`
+	ID                   string `yaml:"id"`
+	EnableTimingGames    bool   `yaml:"enable_timing_games"`
+	TargetFirstRequestMs uint64 `yaml:"target_first_request_ms"`
+	FrequencyGetHeaderMs uint64 `yaml:"frequency_getheader_ms"`
+}
+
+// Config holds all configuration settings from the config file
+type Config struct {
+	TimeoutGetHeaderMs uint64            `yaml:"timeout_get_header_ms"`
+	LateInSlotTimeMs   uint64            `yaml:"late_in_slot_time_ms"`
+	Relays             []RelayConfigYAML `yaml:"relays"`
+}
+
+type ConfigResult struct {
+	RelayConfigs       map[string]types.RelayConfig
+	TimeoutGetHeaderMs uint64
+	LateInSlotTimeMs   uint64
+}
+
+// LoadConfigFile loads configurations from a YAML file
+func LoadConfigFile(configPath string) (*ConfigResult, error) {
+	data, err := os.ReadFile(configPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var config Config
+	if err := yaml.Unmarshal(data, &config); err != nil {
+		return nil, err
+	}
+
+	timeoutGetHeaderMs := config.TimeoutGetHeaderMs
+	if timeoutGetHeaderMs == 0 {
```

Comment:
neeed to clearly specify this in a documentaton on what the default value is. you can do it in the config.yaml